### PR TITLE
- adds change notifications latency information

### LINKED ISSUE #9405: Change notifications: alert does not have latency information
Related #9235 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bc1ec710-fff6-0690-7eb3-16b9773145ef
* Version Independent ID: c4a14ee1-f979-8409-4481-45272b35d965
* Content: [Set up notifications for changes in user data - Microsoft Graph](https://docs.microsoft.com/en-us/graph/webhooks)
* Content Source: [concepts/webhooks.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/concepts/webhooks.md)
* Product: **non-product-specific**
* Technology: **microsoft-graph**
* GitHub Login: @baywet
* Microsoft Alias: **MSGraphDocsVteam**

### PR DESCRIPTION
**Pull request by baywet**
fixes #9405

### CODE REVIEW

File: concepts/webhooks.md [hunk 1]
```diff
@@ -271,6 +271,24 @@ You can optionally configure the firewall that protects your notification URL to
 
 > **Note:** The listed IP addresses that are used to deliver change notifications can be updated at any time without notice.
 
+## Latency
+
+The following table documents the average latency between an event happening in the service and the delivery of the change notification.
```
Comment (baywet) [inline concepts/webhooks.md:276]:
```suggestion
The following table documents the latency to expect between an event happening in the service and the delivery of the change notification.
```

File: concepts/webhooks.md [hunk 2]
```diff
@@ -271,6 +271,24 @@ You can optionally configure the firewall that protects your notification URL to
...
+
+| Resource | Average latency | Maximum latency |
+|:-----|:-----|:-----|
+|[callRecord][] | 15 minutes | 60 minutes |
```
Comment (baywet) [inline concepts/webhooks.md:280]:
```suggestion
|[callRecord][] | below 15 minutes | below 60 minutes |
```

File: concepts/webhooks.md [hunk 3]
```diff
@@ -271,6 +271,24 @@ You can optionally configure the firewall that protects your notification URL to
...
+## Latency
+
+The following table documents the latency to expect between an event happening in the service and the delivery of the change notification.
+
+| Resource | Average latency | Maximum latency |
+|:-----|:-----|:-----|
+|[callRecord][] | below 15 minutes | 60 minutes |
+|[chatMessage][] (preview) | Unknown | Unknown |
+|[contact][] | Unknown | Unknown |
+|[driveItem][] | Unknown | Unknown |
```
Comment (baywet) [inline concepts/webhooks.md:283]:
```suggestion
|[driveItem][] | below 1 minute | 5 minutes |
```

File: concepts/webhooks.md [hunk 4]
```diff
@@ -271,6 +271,24 @@ You can optionally configure the firewall that protects your notification URL to
...
+|[event][] | Unknown | Unknown |
+|[group][] | Unknown | Unknown |
+|[conversation][] | Unknown | Unknown |
+|[list][] | Unknown | Unknown |
```
Comment (baywet) [inline concepts/webhooks.md:287]:
```suggestion
|[list][] | below 1 minute | 5 minutes |
```

File: concepts/webhooks.md [hunk 5]
```diff
@@ -271,6 +271,24 @@ You can optionally configure the firewall that protects your notification URL to
...
 > **Note:** The listed IP addresses that are used to deliver change notifications can be updated at any time without notice.
 
+## Latency
+
+The following table documents the latency to expect between an event happening in the service and the delivery of the change notification.
+
+| Resource | Average latency | Maximum latency |
+|:-----|:-----|:-----|
+|[callRecord][] | below 15 minutes | 60 minutes |
+|[chatMessage][] (preview) | Unknown | Unknown |
```
Comment (baywet) [inline concepts/webhooks.md:281]:
```suggestion
|[chatMessage][] (preview) | below 10 seconds | 1 minute |
```

File: concepts/webhooks.md [hunk 6]
```diff
@@ -271,6 +271,24 @@ You can optionally configure the firewall that protects your notification URL to
...
+The following table documents the latency to expect between an event happening in the service and the delivery of the change notification.
+
+| Resource | Average latency | Maximum latency |
+|:-----|:-----|:-----|
+|[callRecord][] | below 15 minutes | 60 minutes |
+|[chatMessage][] (preview) | below 10 seconds | 1 minute |
+|[contact][] | Unknown | Unknown |
+|[driveItem][] | below 1 minute | 5 minutes |
+|[event][] | Unknown | Unknown |
+|[group][] | Unknown | Unknown |
```
Comment (baywet) [inline concepts/webhooks.md:285]:
```suggestion
|[group][] | below 2 minutes | 15 minutes |
```

File: concepts/webhooks.md [hunk 7]
```diff
@@ -271,6 +271,24 @@ You can optionally configure the firewall that protects your notification URL to
...
+|[conversation][] | Unknown | Unknown |
+|[list][] | below 1 minute | 5 minutes |
+|[message][] | Unknown | Unknown |
+|[alert][] | Unknown | Unknown |
+|[user][] | Unknown | Unknown |
```
Comment (baywet) [inline concepts/webhooks.md:290]:
```suggestion
|[user][] | below 2 minutes | 15 minutes |
```

File: concepts/webhooks.md [hunk 8]
```diff
@@ -271,6 +271,24 @@ You can optionally configure the firewall that protects your notification URL to
...
 > **Note:** The listed IP addresses that are used to deliver change notifications can be updated at any time without notice.
 
+## Latency
+
+The following table documents the latency to expect between an event happening in the service and the delivery of the change notification.
+
+| Resource | Average latency | Maximum latency |
+|:-----|:-----|:-----|
+|[callRecord][] | below 15 minutes | 60 minutes |
+|[chatMessage][] (preview) | below 10 seconds | 1 minute |
```
Comment (baywet) [inline concepts/webhooks.md:281]:
```suggestion
|[chatMessage][] | below 10 seconds | 1 minute |
```

Comment (baywet) [reply concepts/webhooks.md:281]: related to #9372

File: concepts/webhooks.md [hunk 9]
```diff
@@ -271,6 +271,24 @@ You can optionally configure the firewall that protects your notification URL to
...
+|[contact][] | Unknown | Unknown |
+|[driveItem][] | below 1 minute | 5 minutes |
+|[event][] | Unknown | Unknown |
+|[group][] | Unknown | Unknown |
+|[conversation][] | Unknown | Unknown |
+|[list][] | below 1 minute | 5 minutes |
+|[message][] | Unknown | Unknown |
+|[alert][] | Unknown | Unknown |
```
Comment (baywet) [inline concepts/webhooks.md:289]:
```suggestion
|[alert][] | Unknown | Unknown |
|[presence][] (preview) | Unknown | Unknown |
```

File: concepts/webhooks.md [hunk 10]
```diff
@@ -273,6 +273,25 @@ You can optionally configure the firewall that protects your notification URL to
 
 > **Note:** The listed IP addresses that are used to deliver change notifications can be updated at any time without notice.
 
+## Latency
+
+The following table documents the latency to expect between an event happening in the service and the delivery of the change notification.
```
Comment (davidmu1) [inline concepts/webhooks.md:278]:
```suggestion
The following table lists the latency to expect between an event happening in the service and the delivery of the change notification.
```

File: concepts/webhooks.md [hunk 11]
```diff
@@ -273,6 +273,25 @@ You can optionally configure the firewall that protects your notification URL to
...
+
+| Resource | Average latency | Maximum latency |
+|:-----|:-----|:-----|
+|[callRecord][] | below 15 minutes | 60 minutes |
+|[chatMessage][] (preview) | below 10 seconds | 1 minute |
+|[contact][] | Unknown | Unknown |
```
Comment (davidmu1) [inline concepts/webhooks.md:284]: Just curious, does this provide value to list that we don't know what the latency is?

Comment (baywet) [reply concepts/webhooks.md:284]: it's a placeholder for the workloads to fill in. I'm expecting EXO to provide their values within a few weeks, and I'm working with security graph to get theirs as well.

File: concepts/webhooks.md [hunk 12]
```diff
@@ -273,6 +273,25 @@ You can optionally configure the firewall that protects your notification URL to
...
+|[driveItem][] | below 1 minute | 5 minutes |
+|[event][] | Unknown | Unknown |
+|[group][] | below 2 minutes | 15 minutes |
+|[conversation][] | Unknown | Unknown |
+|[list][] | below 1 minute | 5 minutes |
+|[message][] | Unknown | Unknown |
+|[alert][] | Unknown | Unknown |
```
Comment (baywet) [inline concepts/webhooks.md:291]:
```suggestion
|[alert][] | below 3 minutes | 5 minutes |
```

File: concepts/webhooks.md [hunk 13]
```diff
@@ -273,6 +273,25 @@ You can optionally configure the firewall that protects your notification URL to
...
+|[presence][] (preview) | below 10 seconds | 1 minute |
+|[user][] | below 2 minutes | 15 minutes |
+
```
Comment (baywet) [inline concepts/webhooks.md:294]:
```suggestion

>**Note:** the latency provided for the **alert** resource is only applicable once the alert itself has been created. It does not include the time a rule takes to generate an alert from the data.
```

**Comment by baywet** [comment]
fixes #9405

File: concepts/webhooks.md [hunk 14]
```diff
@@ -273,6 +273,26 @@ You can optionally configure the firewall that protects your notification URL to
...
+|[event][] | Unknown | Unknown |
+|[group][] | below 2 minutes | 15 minutes |
+|[conversation][] | Unknown | Unknown |
+|[list][] | below 1 minute | 5 minutes |
+|[message][] | Unknown | Unknown |
+|[alert][] | below 3 minutes | 5 minutes |
+|[presence][] (preview) | below 10 seconds | 1 minute |
+|[user][] | below 2 minutes | 15 minutes |
+
+>**Note:** the latency provided for the **alert** resource is only applicable once the alert itself has been created. It does not include the time it takes for a rule takes to create an alert from the data.
```
Comment (baywet) [inline concepts/webhooks.md:295]:
```suggestion
>**Note:** the latency provided for the **alert** resource is only applicable once the alert itself has been created. It does not include the time it takes for a rule to create an alert from the data.
```

**Pull request merged**